packages/components/src/ChartErrorOverlay.scss

```diff
@@ -0,0 +1,17 @@
+@import '@deephaven/components/scss/custom.scss';
+
+$chart-error-column-valid-color: $green;
+$chart-error-column-invalid-color: $gray-400;
+
+.chart-error-overlay {
+  .chart-error-overlay-content {
+    .btn {
+      margin-top: $spacer-3;
+      margin-right: $spacer-3;
+    }
+    .info-message,
+    .waiting-filter-list {
+      text-align: left;
+    }
+  }
+}
```

Comment:
Should rebase and use CSS variables. We recently switched to using CSS variables and had a lunch 'n' learn on this topic: https://github.com/deephaven-examples/lunch-n-learn/tree/main/css-variables (or https://github.com/deephaven-examples/lunch-n-learn/tree/main/css-variables#stop-using-scss-color-variables)

Variables are here: https://github.com/deephaven/web-client-ui/blob/main/packages/components/src/theme/theme-light/theme-light-semantic.css

(Doesn't look like you're using the SCSS variables declared anyway).